### PR TITLE
refactor(media): Renaming MediaSetting to Settings

### PR DESCRIPTION
**Pull request by ConradJChan**

### CODE REVIEW

File: src/lib/viewers/controls/settings/SettingsAutoplay.tsx [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+import Settings, { Menu } from '.';
 
 export type Props = {
     autoplay: boolean;
     onAutoplayChange: (autoplay: boolean) => void;
 };
 
-export default function MediaSettingsMenuAutoplay({ autoplay, onAutoplayChange }: Props): JSX.Element {
-    const { setActiveMenu } = React.useContext(MediaSettings.Context);
+export default function SettingsMenuAutoplay({ autoplay, onAutoplayChange }: Props): JSX.Element {
```
Comment (jstoffan) [inline src/lib/viewers/controls/settings/SettingsAutoplay.tsx:9]: Let's leave the consumers of the base components in the `media` folder, since they're specific to those viewers.

File: src/lib/viewers/media/MP3.scss [hunk 2]
```diff
@@ -25,6 +25,7 @@
     .bp-media-gear-icon {
         position: absolute;
         right: 10px;
+        width: 36px;
```
Comment (jstoffan) [inline src/lib/viewers/media/MP3.scss:28]: Is this change intentional?

Comment (ConradJChan) [reply src/lib/viewers/media/MP3.scss:28]: Yes, because the `SettingsToggle` no longer uses the `bp-MediaButton` mixin, so the width needs to be adjusted

**Review by jstoffan — APPROVED** [review]
